spec/cobrato_client_ruby_spec.rb

```diff
@@ -1,11 +1,62 @@
 require 'spec_helper'
 
-describe Plugin do
+describe Cobrato do
   it 'has a version number' do
-    expect(Plugin::VERSION).not_to be nil
+    expect(Cobrato::VERSION).not_to be nil
   end
 
-  it 'does something useful' do
-    expect(false).to eq(true)
+  describe ".configuration" do
+    it "is done via block initialization" do
+      Cobrato.configure do |c|
+        c.url = "http://some/where"
+        c.user_agent = "My App v1.0"
+      end
+      expect(Cobrato.configuration.url).to eq "http://some/where"
+      expect(Cobrato.configuration.user_agent).to eq "My App v1.0"
+    end
+
+    it "uses a singleton object for the configuration values" do
+      config1 = Cobrato.configuration
+      config2 = Cobrato.configuration
+      expect(config1).to eq config2
+    end
+  end
+
+  describe ".configure" do
+    it "returns nil when no block given" do
+      expect(Cobrato.configure).to eql(nil)
+    end
+
+    it "raise error if no method" do
+      expect { Cobrato.configure do |c|
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.